CHANGELOG.md

```diff
@@ -1,10 +1,43 @@
-## vNext (TBD)
+## 12.0.0-alpha.0 (2023-03-21)
 
-### Deprecations
-* None
+This is the first pre-release of the next major version of our SDK.
+Please read more and discuss in the dedicated discussion: https://github.com/realm/realm-js/discussions/5416
+
+### Breaking changes
+
+Although this is a complete re-write of our SDK, we've strived to keep breakages to a minimum and expect our users to upgrade from v11 without any notible changes to their code-base.
+
+* In an effort to align with EcmaScript modules, we’re adopting a pattern of named exports on the “root” of the package namespace. This change also affects how our library is imported on CommonJS / Node.js: Before, we exported our constructor directly from the package (`module.export = Realm` style), which would allow CommonJS runtimes like Node.js to import us using a simple assignment:
+```javascript
+const Realm = require(“realm”); // this commonjs style - won’t work anymore
+From now on, users need to consume us using a named import, like this:
```

Comment:
```suggestion
// From now on, users need to consume us using a named import, like this:
```

Reply:
or end the code block and start another after the comment